modules/http4s/src/smithy4s/http4s/SimpleRestJsonBuilder.scala

```diff
@@ -19,12 +19,17 @@ package http4s
 
 import smithy4s.internals.InputOutput
 
-object SimpleRestJsonBuilder
+object SimpleRestJsonBuilder extends SimpleRestJsonBuilder(1024) {
+  def withMaxArity(maxArity: Int): SimpleRestJsonBuilder =
```

Comment:
the method ought to go on the class, not the object.

Reply:
I thought it a bit odd to be on the class ,being that the class has a maxArity parameter in the constructor.

Reply:
Yeah, that's the "fluent builder" pattern. Each `withMethod` is basically equivalent to a `copy` modifying just one field, but has the quality of being much more nicer wrt bincompat.

The point of having the method on the class is that we can add more methods later on if we need to amend the builder to carry more customisation.